app/services/check_answers/property_section.rb

```diff
@@ -21,7 +21,7 @@ def from_session(session_data)
 
     def display_fields
       if owns_property?
-        FIELDS
+        Flipper.enabled?(:partner) ? FIELDS : FIELDS - %i[joint_ownership joint_percentage_owned]
```

Comment:
This is a good spot, but I think it's not enough to check whether the partner flag is enabled, because we also don't want to show these fields if partner flag is enabled _but the client doesn't have a partner_. 

So I think we need to add a `partner` attribute to this model, and make sure that it gets assigned in `#from_session` (maybe `new session_data.slice(*FIELDS + [:partner])`?).

Then this line becomes `partner ? FIELDS : FIELDS - %i[joint_ownership joint_percentage_owned]`

(If the feature flag is not enabled, `partner` will always be false-y).

Reply:
thanks, this is much better and is one less place we need to remove Flipper at a later date